commit 39017b2957fc7f046725339f1914eb7a87db2bea
Author: John Dennis <jdennis@redhat.com>
Date:   Fri Apr 8 09:17:17 2016 -0400

    Fix Content-Type, should be "text/xml" not "application/soap+xml"
    
    There was an error in the example for POST'ing the the AuthnRequest to
    the IdP, the Content-Type header erroneously was
    "application/soap+xml" when in fact it should have been "text/xml".
    
    As background the Content-Type for SOAP 1.1 is "text/xml" but in SOAP
    1.2 it was changed to "application/soap+xml". ECP is specificed to
    used SOAP 1.1.
    
    Signed-off-by: John Dennis <jdennis@redhat.com>

diff --git a/ECP.rst b/ECP.rst
index 4add62a..927ce13 100644
--- a/ECP.rst
+++ b/ECP.rst
@@ -140,7 +140,7 @@ operations.
   response = session.post(ecp.msgUrl,
                           data=ecp.msgBody,
                           auth=requests.auth.HTTPDigestAuth(user, password)
-                          headers={'Content-Type': 'application/soap+xml'})
+                          headers={'Content-Type': 'text/xml'})
 
   # Process returned SOAP wrapped <Assertion> from IdP
   ecp.processResponseMsg(response.text)
